code/__DEFINES/mob_defines.dm

```diff
@@ -276,7 +276,9 @@
 
 #define isnewplayer(A)  (istype((A), /mob/new_player))
 
-#define isorgan(A)		(istype((A), /obj/item/organ/external))
+#define is_external_organ(A)		(istype((A), /obj/item/organ/external))
+#define is_internal_organ(A)		(istype((A), /obj/item/organ/internal))
+#define	isorgan(A)				(istype((A), /obj/item/organ))
```

Comment:
```suggestion
#define	is_organ(A)				(istype((A), /obj/item/organ))
```

Reply:
Forgot that one, good catch! I'll adress this one in VSC